admin/src/app/Components/rooms/rooms.component.html

```diff
@@ -0,0 +1,54 @@
+<form [formGroup]="roomsForm">
+    <table mat-table [dataSource]="rooms">
+        <tr *matHeaderRowDef="tableColumns" mat-header-row></tr>
+        <tr *matRowDef="let row; columns: tableColumns;" mat-row></tr>
+        <ng-container matColumnDef="image">
+            <th mat-header-cell *matHeaderCellDef></th>
+            <td mat-cell *matCellDef="let room">
+                <div class="img__wrap">
+                    <img 
+                        *ngIf="room.mainImageId"
+                        class="img__img" 
+                        [src]="createImagePath(room)"
+                    />
+                    <p class="img__description" (click)="showImagesDialog(room)">
+                        Click to see other images
+                    </p>
+                </div>
+            </td>
+        </ng-container>
+
+        <ng-container matColumnDef="name">
+            <th mat-header-cell *matHeaderCellDef>Room type</th>
+            <td mat-cell *matCellDef="let room">{{room.name}}</td>
```

Comment:
And all other places

```suggestion
            <td mat-cell *matCellDef="let room">{{ room.name }}</td>
```